CONTRIBUTING.md

```diff
@@ -71,7 +89,7 @@ So as to reduce effort for others in understanding a PR or an Issue, authors are
 
 ### Documentation/Acknowledgements
 
-Each new feature or bug fix must be documented in the ChangeLog. Please provide a single-sentence suggestion for that as part of your pull request description. Do not modify the ChangeLog file as part of your PR as it will lead to conflicts.
+Each new feature or bug fix must be documented in the ChangeLog. Please provide a single-sentence suggestion for that as part of your pull request description after the `CHANGELOG: ` line. Do not modify the ChangeLog file as part of your PR as it will lead to conflicts.
```

Comment:
```suggestion
Each new feature or bug fix must be documented in the ChangeLog. Please provide a single-sentence suggestion for that as part of your pull request description after the `CHANGELOG: ` keyword. Do not modify the ChangeLog file as part of your PR as it will lead to conflicts.
```

Minor nit: The description must be in the same line as the script fails to catch it otherwise.